src/KMonad/Keyboard/IO/Windows/Types.hs

```diff
@@ -307,6 +314,271 @@ kcMap = M.fromList $
   -- , (0xFB, ???)             -- Defined as VK_ZOOM
   -- , (0xFC, ???)             -- Defined as VK_NONAME
   -- , (0xFD, ???)             -- Defined as VK_PA1
-  -- , (0xFE, ???)             -- Defined as VK_CLEAR
+  -- , (0xFE, KeyDelete)       -- Defined as VK_OEM_CLEAR
   ]
 
+-- | Translate a KMonad KeyCode to the corresponding Windows virtual-key code
+-- 
+-- We cannot simply reverse the above map for the opposite direction, because
+-- there will be duplicates where more than one virtual-key code produces the
+-- same KMonad KeyCode. See #326
```

Comment:
We should probably put a full link to the relevant GitHub issue here instead of hoping every reader knows which forge we use